### PR TITLE
Content API: Change default ordering to "lft"

### PR DESCRIPTION
**Pull request by manuq**
The API should return the nodes ordered in the same way as they where
authored in Kolibri Studio.

## Reviewer guidance
Create a custom presentation for a channel. Then check that the ordering is the same as the default Kolibri presentation. Which at the same time should be the same ordering as in Kolibri Studio. Here are before/after screenshots:

Before:
regression-before

After:
regression-after

And default (matches after):
regression-default

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/content/api.py [hunk 1]
```diff
@@ -555,12 +555,12 @@ def consolidate(self, items, queryset):
 
 
 class OptionalPagination(ValuesViewsetCursorPagination):
-    ordering = "id"
+    ordering = "lft"
```
Comment (rtibbles) [inline kolibri/core/content/api.py]:
```suggestion
    ordering = ("lft", "id")
```
This seems to have minimal impact on performance and will give a unique ordering every time.

File: kolibri/core/content/api.py [hunk 2]
```diff
@@ -555,12 +555,12 @@ def consolidate(self, items, queryset):
...
     page_size_query_param = "max_results"
 
 
 class OptionalContentNodePagination(OptionalPagination):
-    ordering = "id"
```
Comment (rtibbles) [inline kolibri/core/content/api.py:563]: These two lines don't actually need to be here, can just delete them both, as they are inheriting from above.

**Review by rtibbles — DISMISSED** [review]
Doing the joint ordering seems to be the best solution here.

**Comment by manuq** [comment]
Thanks! Suggestion applied and comment addressed.

**Review by rtibbles — APPROVED** [review]
QA found no regressions, this is good to go.

**Comment by rtibbles** [comment]
Unrelated build failure in the Debian build. Merging.

**Pull request merged**